modules/customizing-the-jenkins-image-stream-tag.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: PROCEDURE
+
+[id="customizing-the-jenkins-image-stream-tag_{context}"]
+= Customizing the Jenkins image stream tag
+
+To control how the Jenkins image is upgraded, you can override the default upgrade behavior by setting the image stream tag value that your Jenkins deployment configurations use.
```

Comment:
In this case, `you can` underlines the optional nature of this task.

Reply:
I have restructured the sentence to emphasize the conditional opening and removed "can" from this phrase.

Reply:
For the abstract, I use a descriptive rather than imperative phrase. In other words, I follow this template:
```
To <summary of task outcome or benefit>, you <summary of action> by using <name of feature>.
```